src/autorest-core/app.ts

```diff
@@ -25,6 +25,10 @@ import { DataStore } from "./lib/data-store/data-store";
 import { EnhancedFileSystem, RealFileSystem } from './lib/file-system';
 import { Exception, OperationCanceledException } from "./lib/exception";
 
+// enable static modules for autorest-core
+if( (<any>global).StaticVolumeSet ) {
```

Comment:
adds it's own static file system if the StaticVolumeSet is in the global namespace.